games-board/sjeng/sjeng-11.2_p8_p1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PV=$(ver_cut 1-2)
+# 11.2_p8_p1 -> 11.2-8.1
+MY_DEB_PV="${MY_PV}-$(ver_cut 4).$(ver_cut 6)"
+
+DESCRIPTION="Console based chess interface"
+HOMEPAGE="http://sjeng.sourceforge.net/"
+SRC_URI="
+	mirror://sourceforge/sjeng/Sjeng-Free-${MY_PV}.tar.gz
+	http://deb.debian.org/debian/pool/main/s/sjeng/sjeng_${MY_DEB_PV}.diff.gz
+"
+
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+IUSE=""
+
+RDEPEND="sys-libs/gdbm:0="
+DEPEND="${RDEPEND}"
+BDEPEND="dev-util/quilt"
+
+S="${WORKDIR}/Sjeng-Free-${MY_PV}"
+
+src_prepare() {
+	default
+
+	eapply "${WORKDIR}/sjeng_${MY_DEB_PV}.diff"
+	QUILT_PATCHES="debian/patches" QUILT_SERIES="debian/patches/series" quilt push -a || die
+	sed -e '/define BOOK/s:".*":"EPREFIX/usr/share/sjeng/":' \
+		-e "s:EPREFIX:${EPREFIX}:" -i book.c || die
+	sed -e '/define RC_GLOBAL/s:".*":"EPREFIX/etc/sjeng.rc":' \
+		-e "s:EPREFIX:${EPREFIX}:" -i rcfile.c || die
```

Comment:
Two stages was to avoid escaping of quotes, because ${EPREFIX} should be in " but " itself is used in the string to be replaced

The source string is `/usr/share/games/sjeng/` so this also was removing `/games/` from it. Do you suggest leaving /games/ there?

Reply:
Yes, that's okay. We stopped pro-actively forcing `/usr/share/games` with `games.eclass` but we don't go out of our way to stop upstream from using it.

Reply:
Alright, done.